albam/engines/mtfw/structs/lmt-67.ksy

```diff
@@ -0,0 +1,66 @@
+meta:
+  endian: le
+  file-extension: lmt
+  id: lmt
+  ks-version: 0.10
+  title: MTFramework animation format
+
+
+seq:
+  - {id: id_magic, contents: [0x4c, 0x4d, 0x54, 0x00]}
+  - {id: version, type: u2}
+  - {id: num_block_offsets, type: u2}
+  - {id: block_offsets, type: block_offset, repeat: expr, repeat-expr: num_block_offsets}
+
+
+types:
+  block_offset:
+    seq:
+      - {id: offset, type: u4}
+    instances:
+      block_header:
+        {type: block_header, pos: offset}
+
+  block_header:
+    seq:
+      - {id: ofs_frame, type: u4}
+      - {id: num_tracks, type: u4}
+      - {id: num_frames, type: u4}
+      - {id: loop_frame, type: u4}
+      - {id: unk_floats, type: f4, repeat: expr, repeat-expr: 8}
+      - {id: unk_00, type: u4}
+      - {id: ofs_buffer_1, type: u4}
```

Comment:
in lmt ver 51 ofs_buffer_1 and ofs_buffer_1 contained some data related to attack animations, but there were also num values for them here. I couldn't find them in ver67 yet but it looks like the animation import worked without them for RE5